src/state/item/item-action-creators.js

```diff
@@ -50,21 +50,7 @@ export function itemFocused(itemIndex) {
                 type: itemActionTypes.ITEM_FOCUSED,
                 data: { itemIndex },
             });
-            const getTags = itemApi.getTags(itemId);
-            const getDescription = itemApi.getDescription(itemId);
-            Promise.all([getTags, getDescription])
-                .then((responses) => {
-                    const { document: { tags } } = responses[0];
-                    const { document: { description } } = responses[1];
-                    dispatch({
-                        type: itemActionTypes.TAGS_UPDATED,
-                        data: { tags },
-                    });
-                    dispatch({
-                        type: itemActionTypes.DESCRIPTION_UPDATED,
-                        data: { description, tempDescription: description },
-                    });
-                });
+            dispatch(fetchItem(itemId));
```

Comment:
Since the `localhost:3000/documents/:doc_id` endpoint now returns metadata fields + tags + description all we need to do is hit this endpoint to get the updated versions of them all.